### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by bnueetheblue** [MeteorDevelopment/meteor-client#1840]
### Describe the bug

https://user-images.githubusercontent.com/96718917/147451949-45d85cfb-3acb-4e38-b647-5a8185db39d6.png

This bug will happen when you look at the item frame, the HUD "looking at" line will describe strange characters instead of block names like other blocks (item frame is not a block but please fix it)

### Steps to reproduce

```bash
Turn on the HUD, turn on the "looking at" line and look at a placed item frame.
```

### Meteor Version

dev build 0.4.7

### Minecraft Version

1.18.1 Fabric 0.12.11

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by SIMULATAN** [comment]
AFAIK, item frames are entities and not "normal blocks".
Because of this, the TargetHUD shows the UUID of the entity, in this case the item frame

**Comment by MineGame159** [comment]
Fixed in 5e383f8e2685545c8111f762687ff7e4cb4025a5.
